tests/end-to-end/simple/App.cs

```diff
@@ -0,0 +1,195 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.S3;
+using Amazon.CDK.AWS.SQS;
+using Constructs;
+using System.Collections.Generic;
+
+namespace Com.Acme.Test.Simple
+{
+    public class NoctStackProps : StackProps
+    {
+        /// <summary>
+        /// The prefix for the bucket name
+        /// </summary>
+        public string BucketNamePrefix { get; set; }
+
+        public string LogDestinationBucketName { get; set; }
+
+    }
+
+    /// <summary>
+    /// An example stack that uses many of the syntax elements permitted in a
+    /// CloudFormation template, but does not attempt to represent a realistic stack.
+    /// </summary>
+    public class NoctStack : Stack
+    {
+        /// <summary>
+        /// The ARN of the bucket in this template!
+        /// </summary>
+        public string BucketArn { get; } 
+
+        /// <summary>
+        /// The ARN of the SQS Queue
+        /// </summary>
+        public string QueueArn { get; } 
+
+        /// <summary>
+        /// Whether this is a large region or not
+        /// </summary>
+        public string IsLarge { get; } 
+
+        NoctStack(Construct scope, string id, NoctStackProps props = null) : base(scope, id, props)
+        {
+            var booleans = new Dictionary<string, Dictionary<string,bool>> 
+            {
+                {
+                    "True", new Dictionary<string, bool>
+                    {
+                            { "true", true },
+                    }
+                },
+                {
+                    "False", new Dictionary<string, bool>
+                    {
+                            { "false", false },
+                    }
+                },
+            };
+            var lists = new Dictionary<string, Dictionary<string,string[]>> 
+            {
+                {
+                    "Candidates", new Dictionary<string, string[]>
+                    {
+                            {
```

Comment:
Is the double indent intentional here? I'm not totally familiar with C# spacing.

Reply:
Thanks for catching! That is not intentional.

Reply:
Fixed